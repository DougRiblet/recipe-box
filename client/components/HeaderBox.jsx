import React from 'react'

const HeaderBox = () => {
  return (
    <div id="recipe-box">
      <h1>Recipe Box</h1>
    </div>
  )
}

export default HeaderBox
