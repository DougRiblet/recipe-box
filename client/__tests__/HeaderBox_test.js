import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import HeaderBox from '../components/HeaderBox'

describe('HeaderBox', () => {
  test('Snapshot', () => {
    const HeaderBox_tree = renderer.create(
      <HeaderBox />
    ).toJSON()
    expect(HeaderBox_tree).toMatchSnapshot()
  })

  test('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<HeaderBox />, div)
  })
})
