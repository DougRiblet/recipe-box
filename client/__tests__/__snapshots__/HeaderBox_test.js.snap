// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderBox Snapshot 1`] = `
<div
  id="recipe-box-header"
>
  <h1>
    Recipe Box
  </h1>
</div>
`;
